src/hostname/hostnamectl.c

```diff
@@ -313,7 +313,7 @@ static int print_status_info(StatusInfo *i) {
                         r = table_add_many(table,
                                            TABLE_FIELD, "Firmware Age",
                                            TABLE_TIMESPAN_DAY, n - i->firmware_date,
-                                           TABLE_SET_COLOR, n - i->firmware_date > USEC_PER_YEAR*2 ? ANSI_HIGHLIGHT_YELLOW : NULL);
+                                           TABLE_SET_COLOR, n - i->firmware_date > USEC_PER_YEAR*2 ? ansi_highlight_yellow() : NULL);
```

Comment:
Note, this is not necessary, as whether color is enabled is checked when displaying tables. But OK.

Reply:
But it is. Please see the corresponding issue.  
When calling `ANSI_HIGHLIGHT_YELLOW` directly the SYSTEMD_COLORS=16 is ignored.